.github/workflows/ci.yaml

```diff
@@ -56,6 +56,10 @@ jobs:
         uses: helm/kind-action@v1.2.0
         if: steps.list-changed.outputs.changed == 'true'
 
+      - name: Create harvester-system namespace
+        run: "kubectl create namespace harvester-system"
+        if: steps.list-changed.outputs.changed == 'true'
```

Comment:
we probably don't need this since the helm test will create the deploy ns automatically.

Reply:
it doesn't seem to. By default it deploys charts to a randomly created namespace.

https://github.com/harvester/charts/actions/runs/5459186904/jobs/9934917079#step:10:2107

seeder chart on initialization is looking to define a `local` cluster object in the harvester-system namespace which is why it fails.